lib/datastax/cloud/index.js

```diff
@@ -198,6 +198,17 @@ function getMetadataServiceInfo(cloudOptions, callback) {
 
 const getMetadataServiceInfoAsync = util.promisify(getMetadataServiceInfo);
 
+async function refreshMetadataServiceInfo(options) {
+
+  if (!options.cloud) {
+    return;
+  }
+  
+  const cloudOptions = new CloudOptions(options);
+  await parseZipFile(cloudOptions);
```

Comment:
This has to be called here (even though we've already processed the zip file) because the current impl of parseZipFile() stores extracted values on the CloudOptions instance.  Since the original version isn't persisted anywhere we have to fully populate the new one we just created before getting metadata info.